### PR TITLE
Release 1.7.1

### PR DESCRIPTION
**Pull request by bowenliang123**
- links on release page , checked ✅
- releases/download page
image

- news
image

- release note:
image

image

### CODE REVIEW

File: src/en/release/1.7.1.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+- [KYUUBI #4331] [KYUUBI #4431] Lineage supports `Union` and `MergeInto`
+
+### Flink Engine
+- [KYUUBI #4666] Support Flink varbinary type in query operation
+- [KYUUBI #4635] Support Flink time type in query operation
+- [KYUUBI #4614] [DOCS] Rename Flink Table Store to Apache Paimon (Incubating) in docs `Connectors for Flink SQL Query Engine`
+- [KYUUBI #4607]  [DOCS] Rename Flink Table Store to Apache Paimon (Incubating) in docs `Connectors for Hive SQL Query Engine`
+- [KYUUBI #4608] [DOCS] Rename Flink Table Store to Apache Paimon (Incubating)  in docs `Connectors for Trino SQL Query Engine`
+- [KYUUBI #4598] Bump Flink-1.15 from 1.15.3 to 1.15.4
+- [KYUUBI #4583] [DOCS] Rename Flink Table Store to Apache Paimon (Incubating)
```
Comment (pan3793) [inline src/en/release/1.7.1.md:126]: We can remove the space between `[LABEL]`s

Comment (bowenliang123) [reply src/en/release/1.7.1.md]: Addressed.

File: hugo.toml [hunk 2]
```diff
@@ -337,6 +337,6 @@ defaultContentLanguage = 'en'
     keepQuotes = true
 
 [params]
-  releases = [ 'v1.7.0', 'v1.6.1-incubating', 'v1.5.2-incubating', 'v1.4.1-incubating', 'v1.3.1-incubating' ]
+  releases = [ 'v1.7.1', 'v1.7.0', 'v1.6.1-incubating', 'v1.5.2-incubating', 'v1.4.1-incubating', 'v1.3.1-incubating']
```
Comment (pan3793) [inline hugo.toml:340]: keep only the latest patched version

File: src/en/release/1.7.1.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+- [KYUUBI #4653][KYUUBI 4650][Improvement] LogDivertAppender supports reading RollingFileAppender pattern
+- [KYUUBI #4623][FOLLOWUP] Fix malformed docs
+- [KYUUBI #4522] `use:catalog` should execute before than `use:database`
+- [KYUUBI #4655][DOCS] Enrich docs for Kyuubi Hive JDBC driver
+- [KYUUBI #4644] Manually terminate the Py4JServer during engine shutdown
+- [KYUUBI #4623][FOLLOWUP] Remove unused import
+- [KYUUBI #4623][K8S] KubernetesApplicationOperation uses Informer instead of list
+- [KYUUBI #3872][BATCH]  Skip setting proxy user when keytab is provided
+- [KYUUBI #4639] Support to specify confOverlay when executing statement with RESTful API
+- [KYUUBI #4627] [Docs] Fix a typo in rest_api.md.
```
Comment (pan3793) [inline src/en/release/1.7.1.md:60]: missed

File: src/en/release/1.7.1.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+- [KYUUBI #4575] Fix the empty last line may causes the session to exit directly
+- [KYUUBI #4568] Make kyuubi-ctl doc enable variable 'release' automatic substitution
+- [KYUUBI #4526][FOLLOWUP] Defer evaluation for GroupProvider
+- [KYUUBI #4519] Update metadata when batch application state changed
+- [KYUUBI #4467][K8S][BATCH] Tolerate Driver Pod ephemerally invisible after submitting
+- [KYUUBI #4496] Call `super.clusterManager()` when `batchConf` does not have `spark.master`
+- [KYUUBI #4494] `bin/kyuubi` should use `exec` to run Kyuubi server
+- [KYUUBI #4491] Fix Trino typo
+- [KYUUBI #4489][REST] Fix missing `org.apache.commons.logging` by including `jcl-over-slf4j` dependency in REST client
+- [KYUUBI #4488] [KSHC] Keep object original name defined in HiveBridgeHelper
```
Comment (pan3793) [inline src/en/release/1.7.1.md:81]: ditto

File: src/en/release/1.7.1.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+- [KYUUBI #4710][ARROW] LocalTableScanExec should not trigger job
+- [KYUUBI #4702][ARROW] CommandResultExec should not trigger job
+- [KYUUBI #4662][ARROW] Arrow serialization should not introduce extra shuffle for outermost limit
+- [KYUUBI #4587][SPARK][EXT] Take care of table cache query stage
+- Spark Authz Plugin
+- [KYUUBI #4585] Authz policy file generation
+- [KYUUBI #4658][Authz] [Bug] Fix InsertIntoHiveDirCommand classname so that we can extract the query in it when authorization.
+- [KYUUBI #4617][AUTHZ] Collect results for filtered show objects ahead to prevent holding unserializable spark plan
+- [KYUUBI #4504][Authz] [Bug] Fix source table privilege requirement when querying permanent view in Spark 3.1 and below
+- [KYUUBI #4532] [AUTHZ] Displays the columns involved in extracting the aggregation operator
```
Comment (pan3793) [inline src/en/release/1.7.1.md:106]: ditto

File: src/en/release/1.7.1.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+- [KYUUBI #4586][TEST][1.7] Fix to clean up views in lineage unit tests
+- [KYUUBI #4525][KSHC] Partitioning predicates should take effect to filter data
+- [KYUUBI #4331][KYUUBI #4431] Lineage supports `Union` and `MergeInto`
+
+### Flink Engine
+- [KYUUBI #4666] Support Flink varbinary type in query operation
+- [KYUUBI #4635] Support Flink time type in query operation
+- [KYUUBI #4614][DOCS] Rename Flink Table Store to Apache Paimon (Incubating) in docs `Connectors for Flink SQL Query Engine`
+- [KYUUBI #4607][DOCS] Rename Flink Table Store to Apache Paimon (Incubating) in docs `Connectors for Hive SQL Query Engine`
+- [KYUUBI #4608] [DOCS] Rename Flink Table Store to Apache Paimon (Incubating)  in docs `Connectors for Trino SQL Query Engine`
```
Comment (pan3793) [inline src/en/release/1.7.1.md:119]: .

File: src/en/release/1.7.1.md [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+- [KYUUBI #4488] [KSHC] Keep object original name defined in HiveBridgeHelper
+- [KYUUBI #4471] Prompted command for docs generation should skip Java tests
+- [KYUUBI #4482] Support to disable batch resource upload function
+- [KYUUBI #4452] Strip the redundant leading and tailing slash of getZooKeeperNamespace.
+- [KYUUBI #4451] Skip etcd test if no docker env
+- [KYUUBI #4433] Fix the case with `create a view from view`
+- [KYUUBI #4464] Simplify and improve log for JDBCMetadataStore
+- [KYUUBI #4462] Fix variable usage issue in `SessionManager#stop`
+- [KYUUBI #4443] Do not set engine session init sql for alive probe session
+- Spark Engine
```
Comment (pan3793) [inline src/en/release/1.7.1.md:90]: wrong level

File: src/en/release/1.7.1.md [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+- [KYUUBI #4749] Fix flaky test issues in SchedulerPoolSuite
+- [KYUUBI #4722][DOCS] Kyuubi requires Spark 3.1 and above now
+- [KYUUBI #4720][ARROW] Fix java.lang.NoSuchFieldError: IpcOption.DEFAULT for Spark-3.1/3.2
+- [KYUUBI #4713][TEST] Fix false positive result in SchedulerPoolSuite (#4714)
+- [KYUUBI #4713][TEST] Fix false positive result in SchedulerPoolSuite
+- [KYUUBI #4710][ARROW] LocalTableScanExec should not trigger job
+- [KYUUBI #4702][ARROW] CommandResultExec should not trigger job
+- [KYUUBI #4662][ARROW] Arrow serialization should not introduce extra shuffle for outermost limit
+- [KYUUBI #4587][SPARK][EXT] Take care of table cache query stage
+- Spark Authz Plugin
```
Comment (pan3793) [inline src/en/release/1.7.1.md:101]: wrong level

File: src/en/release/1.7.1.md [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+- [KYUUBI #4585] Authz policy file generation
+- [KYUUBI #4658][Authz] [Bug] Fix InsertIntoHiveDirCommand classname so that we can extract the query in it when authorization.
+- [KYUUBI #4617][AUTHZ] Collect results for filtered show objects ahead to prevent holding unserializable spark plan
+- [KYUUBI #4504][Authz] [Bug] Fix source table privilege requirement when querying permanent view in Spark 3.1 and below
+- [KYUUBI #4532] [AUTHZ] Displays the columns involved in extracting the aggregation operator
+- [KYUUBI #4450] Ignore unknown fields `policyPriority` when reading policy json file
+- Other Spark Extensions
```
Comment (pan3793) [inline src/en/release/1.7.1.md:108]: wrong level

**Comment by pan3793** [comment]
The news picture in PR description is broken

**Comment by bowenliang123** [comment]
> The news picture in PR description is broken

Updated.

**Review by pan3793 — APPROVED** [review]

**Comment by yaooqinn** [comment]
Would you please keep the style we use for release notes?

**Pull request merged**